machine/machine_core/constants.py

```diff
@@ -3,8 +3,8 @@
 from lib.constants import OSTREE_IMAGES, MACHINE_DIR, BOTS_DIR, BASE_DIR, TEST_DIR, GIT_DIR, \
     IMAGES_DIR, SCRIPTS_DIR, DEFAULT_IDENTITY_FILE, TEST_OS_DEFAULT, DEFAULT_IMAGE
 
-__all__ = (OSTREE_IMAGES, BOTS_DIR, MACHINE_DIR, BASE_DIR, TEST_DIR, GIT_DIR,
-           IMAGES_DIR, SCRIPTS_DIR, DEFAULT_IDENTITY_FILE, TEST_OS_DEFAULT, DEFAULT_IMAGE,)
+__all__ = ("OSTREE_IMAGES", "BOTS_DIR", "MACHINE_DIR", "BASE_DIR", "TEST_DIR", "GIT_DIR",
```

Comment:
Did we decide to kill off all uses of `__all__` or not?

Reply:
Not yet, we can discuss if we want to do this everywhere.

Reply:
Found out yesterday, it's all over Python stdlib.